Week1/homework/JS-Exercises/Exercise-05-Round a number and log it.js

```diff
@@ -0,0 +1,17 @@
+let z = 7.25;
+console.log("z " + "= " + z);
+
+let a = Math.round(z);
+console.log("a " + "= " + a);
+
+let max;
+
+if(z > a) {
+    max = z;
+    console.log("The height is z." + " " + "z = " + z)
+};
+
+if( a > z) {
+    max = a;
+    console.log("The height is a." + " " + "a = " + a)
+};
```

Comment:
Couple things here:
1. You could use `const` instead of `let` for variables `a` and `z` since you don't reassign them anywhere in your program.
2. You could also try to implement it using conditional operators as this is a very good case of using them.